src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -72,17 +74,18 @@ class DBS3Reader(object):
     General API for reading data from DBS
     """
 
-    def __init__(self, url, logger=None, **contact):
+    def __init__(self, url, logger=None, parallel=None, **contact):
```

Comment:
Can you please create a docstring for this class and specify what `parallel` is for?

Reply:
done